System.IO.Packaging/FileFormatException.cs

```diff
@@ -27,7 +25,7 @@ public class FileFormatException : FormatException
         /// This message takes into account the current system culture.
         /// </summary>
         public FileFormatException()
```

Comment:
Why do we implement the `System.IO.FileFormatException` at all? Isn't this already provided by other parts of the .NET Framework, also as part of `System.IO` namespace?